packages/loading-screen/src/main.js

```diff
@@ -11,7 +11,7 @@ import LoadingRetry from '@/views/LoadingRetry.vue';
 import store from "@/store";
 
 // Language codes are RFC 5646 (https://datatracker.ietf.org/doc/html/rfc5646)
-const supportedLocales = ['en-US'];
+const supportedLocales = ['en-US', 'es-419'];
```

Comment:
I'm kind of doubtful that we'll ever have region specific translations and instead maybe these should just be `en` and `es`. But I don't know if that hoses the runtime locale selection. In chrome for me, `navigator.languages` is `['en-US', 'en']`. That can be punted on, regardless.

Reply:
That would hose the runtime locale selection. Maybe we’ll want to make that selection code cleverer in future (for example, selecting `es` if the user asks for `es-MX` but no `es-MX` translation exists), but for now this will probably do.